### PR TITLE
Feature/organization collection permissions

### LINKED ISSUE #8: Create organization collection permissions
_From @bajiat on February 19, 2016 12:16_

Define who can add, edit (and delete) organizations. Create roles and permission for these actions.

Definition of done
---
* [ ] Administrator id(s) field exists in organization schema.
* [ ] Administrator id(s) field type is an array of string.
* [ ] When adding an organization, user id is added to administrator id's field in organization document.
* [ ] User gets assigned an administrator role.
* [ ] Admininistrator role is allowed to edit / update organization.

_Copied from original issue: Dipor/platform#11_

### PR DESCRIPTION
**Pull request by brylie**
Closes #8
# Proposed changes
- allow logged in users to create organization
- allow system administrator to update or delete organizations
- allow organization administrator to update or delete organizations

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

**Comment by bajiat** [comment]
@frenchbread Would you be available for reviewing this PR?

**Comment by frenchbread** [comment]
@bajiat yes

**Comment by frenchbread** [comment]
Good use of `ongoworks:security` package!

File: organizations/collection/permissions.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// Logged in user may create organization
+Organzations.permit('insert').apply();
+
+// System administrator role may update or delete
+Organizations.permit(["update", "delete"]).ifHasRole("admin").apply();
```
Comment (jykae) [inline organizations/collection/permissions.js]: @brylie @frenchbread getting error from this line:
TypeError: Object [object Object] has no method 'ifHasRole'
W20160302-12:59:25.260(2)? (STDERR)     at organizations/collection/permissions.js:5:44

**Pull request merged**